process-pr-stats

```diff
@@ -135,11 +134,17 @@ if __name__ == "__main__":
       isPr = issue.pull_request and 1 or 0
       updateAt = int(mktime(datetime.strptime(str(issue.updated_at), "%Y-%m-%d %H:%M:%S").timetuple()))
 
+      if (issue.milestone is not None):
+        milestone = str(issue.milestone.title)
+      else:
+        milestone = "-"
+
```

Comment:
I usually prefer to write this as:

milestone = issue.milestone and str(issue.milestone.title) or "-"

(if it stays  below 80 chars)